### PR TITLE
networking: Use links instead of buttons to open dialogs

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

File: test/verify/check-networking [hunk 1]
```diff
@@ -120,7 +120,7 @@ class TestNetworking(MachineCase):
 
         # Configure a manual IP address
         #
-        b.click("tr:contains('IPv4') button")
+        b.click("tr:contains('IPv4') a", True)
```
Comment (mvollmer) [inline test/verify/check-networking]: I had to use 'force' here because phantomjs doesn't consider the link to be visible.  Does anyone know what's going on?

Comment (stefwalter) [inline test/verify/check-networking]: I would put a ```console.log("click visible", sel, el.offsetWidth, el.offsetHeight)``` in ph_click() where we check the offsetWidth and offsetHeight. I've had to tweak this once before, so there's a chance that it doesn't account for certain corner cases.

Comment (mvollmer) [inline test/verify/check-networking]: I did that, and the offsetWidth and offsetHeight stay zero...

**Comment by stefwalter** [comment]
Looks like there's a related failure:

```
Page error: tr:contains('Ethernet') button not found
phantomjs://code/phantom-lib.js 24 ph_only
phantomjs://code/phantom-lib.js 33 ph_find
phantomjs://code/phantom-lib.js 104 ph_click
 1 
 1 
```

File: test/verify/check-networking (hunk 1 above)
Comment (mvollmer) [inline test/verify/check-networking]: The `a` element had illegal content, actually: a couple of `div`s.  I have fixed that.

**Comment by andreasn** [comment]
UI looks good.
